packages/arb-token-bridge-ui/src/hooks/TransferPanel/useGasSummaryStore.ts

```diff
@@ -0,0 +1,237 @@
+import { BigNumber, constants, utils } from 'ethers'
+import { create } from 'zustand'
+import { useAccount } from 'wagmi'
+import { useCallback, useEffect, useMemo, useState } from 'react'
+
+import { TransferPanelSummaryToken } from '../../components/TransferPanel/TransferPanelSummary'
+import { useNetworksAndSigners } from '../useNetworksAndSigners'
+import { useAppState } from '../../state'
+import { useGasPrice } from '../useGasPrice'
+import { useDebouncedValue } from '../useDebouncedValue'
+import {
+  isTokenArbitrumGoerliNativeUSDC,
+  isTokenArbitrumOneNativeUSDC
+} from '../../util/TokenUtils'
+import { withdrawTokenEstimateGas } from '../../util/TokenWithdrawalUtils'
+import { withdrawEthEstimateGas } from '../../util/EthWithdrawalUtils'
+import { depositEthEstimateGas } from '../../util/EthDepositUtils'
+import { depositTokenEstimateGas } from '../../util/TokenDepositUtils'
+
+const INITIAL_GAS_ESTIMATION_RESULT: GasEstimationResult = {
+  // Estimated L1 gas, denominated in Wei, represented as a BigNumber
+  estimatedL1Gas: constants.Zero,
+  // Estimated L2 gas, denominated in Wei, represented as a BigNumber
+  estimatedL2Gas: constants.Zero,
+  // Estimated L2 submission cost is precalculated and includes gas price
+  estimatedL2SubmissionCost: constants.Zero
+}
+
+const INITIAL_GAS_SUMMARY_RESULT: UseGasSummaryResult = {
```

Comment:
Any reason for gas estimation being stored as BigNumber and gas summary as number?

Reply:
hm, it was just how it was before this PR
didn't change it